### ISSUE TITLE
30 FPS, No matter what

### ISSUE THREAD

**Issue by thirtyought6** [supraball/issues#64]
hello, i have a stupid problem, help me solve it.

specs = i7 3770k, Geforce 670, 16g Ram, windows 7 64 bit home premium

problem = i always have 30 or below FPS

what i have a tried = every different setting in the updated video options, reinstalling, running as admin, high priority, VSYNC IS NOT ON, the performance thread at reddit tried all the stuff in there, standing directly in front of a wall in Hall still 30 fps.

i know i should be having A LOT more FPS than this, any help is appreciated even the obvious stuff that you thought of that i haven't thought of. Please help me not play in Peasant box mode....

**Comment by Timoses** [comment]
Did you also try changing the `Max Framerate` option in the visuals tab?

**Comment by DavidM1337** [comment]
I once was limited to 100 FPS even though I disabled the fps limiter completely and the limit was 120 or so. I have no idea why, but maybe it's the same case here, just a with a limit of 30.

**Comment by thirtyought6** [comment]
Yes, I've tried changing max framer ate option to above and below 30, no changes.
